README.md

```diff
@@ -10,6 +10,9 @@ This section has projects that have just a basic UI and simple logic with locali
 
 ## Level-2: Intermediate
 - [GoogleBlessing-App](https://github.com/uzairleo/GoogleBlessings)
+- [Top10-Flutter-Widget](https://github.com/uzairleo/top10_flutter_widget)
+- [Flutter-SplashScreen-Approach](https://github.com/uzairleo/flutter_splashscreen_tutorial)
```

Comment:
This is a basic widget, so please add it to the basic section.

Reply:
sorry plz it was a mistake. i will correct it in couple of minutes

Reply:
Add both apps to the basic category

Reply:
yes i did that with new pull request